### ISSUE TITLE
Connector doesn't appear in data sources

### ISSUE THREAD

**Issue by akzincsystems** [CreativeDataEU/PowerBITrinoConnector#34]
Followed your instructions in the readme + those at https://learn.microsoft.com/en-us/power-bi/connect-data/desktop-connector-extensibility.  I noticed that the .mez file had a block on it, in its properties page, so removed that.

Not seeing the connector in data sources.

PBI Desktop V 2.138.1420.0 64-bit (November 2024)

have I missed something or has PBI changed something that you know of?

Thanks

**Comment by pichlerpa** [comment]
No, not that we know of. You could try to open Power BI Desktop as Admin to see if there are any problems with access rights.

**Comment by pichlerpa** [comment]
Closed due to inactivity.
